risc0/zkvm/methods/guest/build.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2025 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use k256::ecdsa::{signature::Signer as _, Signature, SigningKey, VerifyingKey};
+use std::fmt::Write as _;
+use std::path::Path;
```

Comment:
```suggestion
use std::{fmt::Write as _, path::Path};

use k256::ecdsa::{signature::Signer as _, Signature, SigningKey, VerifyingKey};
```

This is some of the formatting that we used when nightly was in use.

Reply:
okay thanks, good to know